rollup.config.js

```diff
@@ -1,18 +1,38 @@
-export default {
-	input: './src/index.js',
-	treeshake: false,
-	external: p => /^three/.test( p ),
-
-	output: {
-
-		name: 'MeshBVHLib',
-		extend: true,
-		format: 'umd',
-		file: './umd/index.js',
-		sourcemap: true,
-
-		globals: p => /^three/.test( p ) ? 'THREE' : null,
-
+export default [
+	{
+		input: './src/index.js',
+		treeshake: false,
+		external: p => /^three/.test( p ),
+	
+		output: {
+	
+			name: 'MeshBVHLib',
+			extend: true,
+			format: 'umd',
+			file: './umd/index.js',
+			sourcemap: true,
+	
+			globals: p => /^three/.test( p ) ? 'THREE' : null,
+	
+		},
+	
 	},
-
-};
+	{
+		input: './src/index.js',
+		treeshake: false,
+		external: p => /^three/.test( p ),
+	
+		output: {
+	
+			name: 'MeshBVHLib',
+			extend: true,
+			format: 'esm',
+			file: './esm/index.js',
+			sourcemap: true,
+	
+			globals: p => /^three/.test( p ) ? 'THREE' : null,
```

Comment:
For the "esm" build I don't think the `name`, `extend`, or `global` options are necessary -- can we remove those?

Reply:
Removed and tested that the output is the same!